apps/force/src/utils/index.ts

```diff
@@ -0,0 +1,230 @@
+import { gql, request } from "graphql-request";
```

Comment:
In general would suggest adding unit test for non trivial 'util' functions. personally i like splitting 'pure' function util functions collections from functions that are dependant on side effects ( in this case the backend ), consider moving data fetching functions to separate module or caller.